SignalMessaging/categories/UIColor+OWS.h

```diff
@@ -31,12 +31,10 @@ NS_ASSUME_NONNULL_BEGIN
 #pragma mark - ConversationColor
 
 + (nullable UIColor *)ows_conversationColorForColorName:(NSString *)colorName
-    NS_SWIFT_NAME(ows_conversationColor(colorName:));
-+ (nullable NSString *)ows_conversationColorNameForColor:(UIColor *)color
-    NS_SWIFT_NAME(ows_conversationColorName(color:));
+                                               isShaded:(BOOL)isShaded
+    NS_SWIFT_NAME(ows_conversationColor(colorName:isShaded:));
```

Comment:
We could have the impl check for dark theme, but I thought it'd be good to force the call site to specify whether or not the color should be shaded.

Reply:
Yeah - I prefer this rather than have UIColor depend on `Theme` state implicitly.